_episodes/03-monitor-the-model.md

```diff
@@ -729,15 +723,105 @@ plt.ylabel("true sunshine hours")
 ~~~
 {: .language-python}
 
-![Output of plotting sample](../fig/03_regression_test_5_dropout_batchnorm.png)
+![Output of plotting sample](../fig/03_regression_test_5_dropout_batchnorm.png){: width="500px"}
 
 Well, the above is certainly not perfect. But how good or bad is this? Maybe not good enough to plan your picnic for tomorrow.
 But let's better compare it to the naive baseline we created in the beginning. What would you say, did we improve on that?
 
+> ## Exercise: Simplify the model and add data
```

Comment:
I like this exercise, because it lets the learner repeat the whole process from selecting data up to traning and evaluating a model.
Two comments:
- In the solution, give a hint on what you could see in the results
- I haven't tried running this code yet, but is the runtime approximately the same or will it take much longer to train (because of more training data, although you have fewer features)? If so, it would be good to add a warning here about training time.

Reply:
thanks @dafnevk for feedback. I added more verbosity and descriptive text in the solution about what to expect. The training time goes down when removing non-Basel cities even with 3x more data because the number of features is reduced by ~9x

Reply:
Sorry nag here, the code uses a variable `earlystopper`. It is not defined anywhere as far as I can see.

Reply:
fixes landed in https://github.com/carpentries-incubator/deep-learning-intro/pull/196/commits/00dbbe9dff074ded627c2e4e4e71f933c68dcb7e